README.md

```diff
@@ -13,7 +13,7 @@ This project is built using [Antora](https://antora.org/).
 If you would like to contribute to the TinyMCE project please read the TinyMCE Documentation Contributor’s Guide at either:
 
 - [TinyMCE Documentation - Contributor's Guide](https://www.tiny.cloud/docs/configure/contributing-docs/).
-- [GitHub - Contributing to the TinyMCE developer documentation](https://github.com/tinymce/tinymce-docs/blob/develop/CONTRIBUTING.md#contributing-to-the-tinymce-developer-documentation).
+- [GitHub - How to contribute to TinyMCE’s documentation](https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/CONTRIBUTING.md#how-to-contribute-to-tinymces-documentation).
```

Comment:
@FarzadHayat we should point this link to `release` not `staging` for public pull requests.

I've added the link, but check confirm on your side that the link and anchor works as expected.

```suggestion
- [GitHub - How to contribute to TinyMCE’s documentation](https://github.com/tinymce/tinymce-docs/blob/release/docs-6/CONTRIBUTING.md#how-to-contribute-to-tinymces-documentation).
```